checkov/terraform/checks/resource/aws/BackupVaultEncrypted.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.consts import ANY_VALUE
+
+
+class BackupVaultEncrypted(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Backup Vault server-side encryption is enabled."
```

Comment:
the text is actually misleading, because AWS Backup will use an AWS managed KMS key, if you don't set it. So something more like
`Ensure Backup Vault is encrypted at rest using KMS CMK`